packages/gatsby-plugin-image/graphql.js

```diff
@@ -0,0 +1 @@
+export * from "./dist/resolver-utils"
```

Comment:
I've moved the graphql helper into a separate subpackage. Having them in the main gatsby-plugin-image was causing build problems, as it was trying to import graphql it when building SSR pages, causing build failures. Moving it into `"gatsby-plugin-image/graphql"` fixes that. I'm guessing this is the reason gatsby re-exports graphql from a subpackage too?